pages/dkp/konvoy/2.3/choose-infrastructure/eks/advanced/eks-cluster-iam-polices-and-roles/index.md

```diff
@@ -0,0 +1,226 @@
+---
```

Comment:
I don't think we want to add this page to the 2.3 github repo;  The 2.3 work on this is being done in confluence so I don't think we want it here also.

Reply:
True Craig

Reply:
Not a huge deal if it gets left in, but definitely need the word policies spelled right in the title from branch.  I would pull and checkout your branch, but I REALLY don't want to f-this PR up.  So, I'll wait until you have a chance Faiq.